src/org/gk/qualityCheck/HumanReactionsWithNonHumanPhysicalEntitiesWithoutDiseaseCheck.java

```diff
@@ -0,0 +1,100 @@
+package org.gk.qualityCheck;
+
+import org.gk.database.InstanceListPane;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaClass;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+/**
+ * Flags all human Reactions that contain Non-only-Human PhysicalEntities that do not have a populated disease attribute.
```

Comment:
I was a bit unsure of the term "Non-only-Human".  Does it mean Physical Entities that have at least one constituent part that is non-human?

Reply:
It's meant to convey either non-human species or non-populated species, I believe. I'll make that clearer (and less hypheny)